packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -33,9 +36,8 @@ export default class HdsAppSideNav extends Component<HdsAppSideNavSignature> {
   private _desktopMQ: MediaQueryList;
   private _containersToHide!: NodeListOf<Element>;
 
-  private _desktopMQVal = getComputedStyle(
-    document.documentElement
-  ).getPropertyValue('--hds-app-desktop-breakpoint');
+  // we use the `lg` breakpoint for `desktop` viewports, but consumers can override its value
+  private _desktopMQVal = this.args.breakpoint ?? hdsBreakpointsValues['lg'].px;
```

Comment:
This technically is a breaking change. That said, the `AppSideNav` is not yet adopted, and [there are no usage](https://github.com/search?q=%22--hds-app-desktop-breakpoint%22+user%3Ahashicorp&type=code&ref=advsearch) of `--hds-app-desktop-breakpoint` in any of the codebases (apart from our monorepo), so unless there are objections I would like to pass this change as `minor`.

Reply:
Following discussion in the HDS Engineering sync, we've agreed it's OK to release it as a `minor` changeset.